logstash-core/logstash-core.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
   gem.version       = LOGSTASH_CORE_VERSION
 
-  gem.add_runtime_dependency "logstash-core-event-java", "~> 3.0.0.dev"
+  gem.add_runtime_dependency "logstash-core-event-java", "2.3.0.snapshot1"
```

Comment:
@colinsurprenant is that the correct constraints? `~>` version strict? I believe the strict is the way to go just wanted to make sure.